apex_checks.xml

```diff
@@ -97,6 +97,10 @@
       <message key="ws.notPreceded" value="WhitespaceAround: ''{0}'' is not preceded with whitespace."/>
     </module>
 
+    <module name="LineLength">
+      <property name="max" value="120"/>
```

Comment:
This was agreed in discussion on the mailing list.



Thank you,
Ilya Ganelin



-----Original Message-----
From: Chetan Narsude [notifications@github.com<mailto:notifications@github.com>]
Sent: Wednesday, November 11, 2015 12:10 AM Eastern Standard Time
To: apache/incubator-apex-core
Cc: Ganelin, Ilya
Subject: Re: [incubator-apex-core] [APEX-230] Set line length to 120 (#156)


In apex_checks.xml<https://github.com/apache/incubator-apex-core/pull/156#discussion_r44502182>:


> @@ -97,6 +97,10 @@
>        <message key="ws.notPreceded" value="WhitespaceAround: ''{0}'' is not preceded with whitespace."/>
>      </module>
>
> +    <module name="LineLength">
> +      <property name="max" value="120"/>


what's the source of this magic number? why not 80, or 100 or 150?

—
Reply to this email directly or view it on GitHub<https://github.com/apache/incubator-apex-core/pull/156/files#r44502182>.
________________________________________________________

The information contained in this e-mail is confidential and/or proprietary to Capital One and/or its affiliates and may only be used solely in performance of work or services for Capital One. The information transmitted herewith is intended only for use by the individual or entity to which it is addressed. If the reader of this message is not the intended recipient, you are hereby notified that any review, retransmission, dissemination, distribution, copying or other use of, or taking of any action in reliance upon this information is strictly prohibited. If you have received this communication in error, please contact the sender and delete the material from your computer.